### ISSUE TITLE
Missing parameters in finetuning script

### ISSUE THREAD

**Issue by FSet89** [LLaVA-VL/LLaVA-NeXT#209]
I want to finetune the 7b model on my dataset, possibly using lora. How can I do it? I tried the finetune_onevision.sh script but I don't know how to set some parameters (rank, port, nnodes, addr, mid_run_name). I am training on a single machine.

**Comment by choiszt** [comment]
Hi, have you solve this problem?

**Comment by FSet89** [comment]
No, I switched to [LLaVA](https://github.com/haotian-liu/LLaVA) where I didn't encounter it. However, I hope they fix it

**Comment by FSet89** [comment]
Yes, I'm using that repo until this problem is identified/fixed

**Comment by choiszt** [comment]
I have set `export NCCL_P2P_DISABLE=1  
export NCCL_IB_DISABLE=1   
export NCCL_SOCKET_IFNAME=eth0  
export NCCL_SHM_DISABLE=1 `
It temporarily works to me @FSet89

**Comment by OBJECT907** [comment]
> I have set `export NCCL_P2P_DISABLE=1 export NCCL_IB_DISABLE=1 export NCCL_SOCKET_IFNAME=eth0 export NCCL_SHM_DISABLE=1 ` It temporarily works for me @FSet89

Not work for me.

**Comment by lxr-1204** [comment]
I have solved this issue. For me, this problem was another form of OOM (Out of Memory), and you can solve it by addressing the OOM itself. For example, by adding more GPUs or enabling LoRA. Specifically, you can also reduce the max_length, but this may cause token truncation, so please adjust it based on your dataset.
Good luck!

**Comment by MinusOne2** [comment]
just comment this...
```
export OMP_NUM_THREADS=8
export NCCL_IB_DISABLE=0
export NCCL_IB_GID_INDEX=3
export NCCL_SOCKET_IFNAME=eth0
export NCCL_DEBUG=INFO
